config/routes.rb

```diff
@@ -49,6 +49,56 @@
   #get  '/sosna/solution/:id/down(/:ori)'     => 'sosna/solution#download',      :as => :sosna_solution_download
   #get  '/sosna/solution/:id/down_rev(/:ori)'=> 'sosna/solution#download_rev', :as => :sosna_solution_download_rev
 
+  # scouts:
```

Comment:
Scout se mi vazne nelibi. Nemohl byt to byt `attendant` nebo `paticipant`?

Reply:
```participant``` je hloupé, protože to už je takové víceméně zabrané ve formě ```event_participant```, neboli něco, co ti říká, že se daný člověk účastní dané akce.

Přijde mi, že chceme mít nějaký ekvivalent ``solver```a pro akce, což ```attendant``` může pořád znamenat hromadu věcí. Oproti tomu ```scout``` ti říká, že venkonvní akce jsou tím jediným důvodem, proč by ten účet měl existovat

Uživatelé to vidí jako 'účastnický účet', což je ale moc dlouhé na to, aby se to zkonvertovalo do kódu přímo

Reply:
Podle me je to zavadejici oznaceni. Scaut je nejaky pruzkumnik, spion nebo donasec. Navic se mi scout nelibi jako samo o sobe pro jeho spojeni s divnou americkou detskou organizaci (z ceho si musim uznat cesky junak vybral ty lepsi kousky).

Reply:
ok, místo "scout" se to jmenuje "participant"